pkg/support/dump_windows_test.go

```diff
@@ -0,0 +1,101 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build windows
+// +build windows
+
+package support
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/spf13/afero"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"k8s.io/utils/exec"
+	exectesting "k8s.io/utils/exec/testing"
+)
+
+type testExec struct {
+	exectesting.FakeExec
+}
+
+// Command func is to track correct commands are executed
+func (te *testExec) Command(cmd string, args ...string) exec.Cmd {
+	fakeCmd := new(exectesting.FakeCmd)
+	fakeCmd.CombinedOutputScript = append(fakeCmd.CombinedOutputScript, func() ([]byte, []byte, error) {
+		return []byte(fmt.Sprintf("%s %s", cmd, strings.Join(args, " "))), nil, nil
+	})
+	return fakeCmd
+}
+
+func TestDumpLog(t *testing.T) {
+	basedir := "dir1/dir2"
+	fs := afero.NewMemMapFs()
+	dumper := NewAgentDumper(fs, nil, nil, nil, nil, "7s", true, true)
+	err := dumper.DumpLog(basedir)
+	assert.NoError(t, err)
```

Comment:
the test is not checking for anything besides making sure that no error is returned?
shouldn't we check the contents of the file system after the call to `DumpLog`?